model/task/task.go

```diff
@@ -2514,13 +2515,22 @@ func (t *Task) Insert() error {
 // into the old_tasks collection. If this is a display task, its execution tasks
 // are also archived.
 func (t *Task) Archive() error {
+	if !utility.StringSliceContains(evergreen.TaskCompletedStatuses, t.Status) {
+		grip.Debug(message.Fields{
```

Comment:
Is this logging meant to be temporary or permanent?

Reply:
I put this here initially for my staging testing but I think it'd be good to monitor post deploy, since it should allow us to better identify future attempts of the system to reset unfinished tasks and we can address that cause. 

I can create a ticket for removing once we determine it's not helpful anymore

Reply:
Yeah that sounds reasonable -- a lot of times as part of the debug statement we include the ticket for removing/tracking it  ("ticket": <ticket_num>) if that's helpful (since this also helps with checking on the debug statements)